electron/gog/games.ts

```diff
@@ -335,7 +335,11 @@ class GOGGame extends Game {
       let wineFlag = ['--wine', wineVersion.bin]
       let winePrefixFlag = ['--wine-prefix', winePrefix]
       if (wineVersion.type === 'proton') {
-        wineFlag = ['--no-wine', '--wrapper', `'${wineVersion.bin}' run`]
+        const runtime = getSteamRuntime()
+        const runWithRuntime = runtime.path
```

Comment:
Maybe a `logWarning` if a runtime isn't found would be good

Reply:
Added the warning now